components/global.test.js

```diff
@@ -0,0 +1,113 @@
+import { shallow } from "enzyme";
+import   {MyApp } from "../pages/_app"
+import AppHead from "../Utils/layouts/head";
+import { PageFooter} from "../components/footer"
+import BreadCrumb, {BreadAnchor, BreadLink} from "../Utils/layouts"
+import * as nextRouter from 'next/router';
+import { findByTestAtrr} from "../Utils/test";
+
+
+describe("AppJs", () => {
+
+  let app;
+    
+      
+  beforeAll(() => {
+    app = shallow(<MyApp />);
+  });
+
+  afterAll(() => {
+    app.unmount();
+  });
+
+    it("renders footer component", () => {
+        
+        const component = findByTestAtrr(app, "contentHead");
+        expect(component.length).toBe(1);
+          
+    })
+    it("renders head component", () => {
```

Comment:
You have the wrong test description here, should be `renders footer component` and on line 23 should be `renders head component`.

Reply:
👍 Thanks. Sorted